packages/automated-dispute/tests/protocolProvider.spec.ts

```diff
@@ -242,11 +264,11 @@ describe("ProtocolProvider", () => {
                 mockedPrivateKey,
             );
 
-            const mockRequest = {} as Request["prophetData"];
-            const mockResponse = {} as Response["prophetData"];
+            const mockRequestProphetData = DEFAULT_MOCKED_REQUEST_CREATED_DATA.prophetData;
+            const mockResponseProphetData = DEFAULT_MOCKED_RESPONSE_DATA.prophetData;
```

Comment:
No need to change this in this PR, just as a side note.

It might be better to use the `mocks.buildResponse(request)` helper function here to make it clear that there's a link between the response and the request that you're using.

```suggestion
            const mockRequest = DEFAULT_MOCKED_RESPONSE_DATA;
            const mockResponse = mocks.buildResponse(mockRequest);
            
            await expect(
                protocolProvider(mockRequest.prophetData, mockResponse.prophetData)
            )
```

[Here](https://github.com/defi-wonderland/EBO-agent/blob/2d1de10fc927fff1f160bcec017bf8873306eab1/packages/automated-dispute/tests/services/eboActor/onResponseDisputed.spec.ts#L13) you've got that helper function in action.